tests/sentry/web/frontend/test_auth_organization_login.py

```diff
@@ -652,3 +654,70 @@ def test_flow_as_unauthenticated_existing_user_legacy_identity_migration(self):
         # Ensure the ident was migrated from the legacy identity
         updated_ident = AuthIdentity.objects.get(id=user_ident.id)
         assert updated_ident.ident == "foo@new-domain.com"
+
+    @override_settings(SENTRY_SINGLE_ORGANIZATION=True)
+    @with_feature({"organizations:create": False})
+    def test_basic_auth_flow_as_invited_user(self):
```

Comment:
this is the money test, as before, with this value in `_next`, we'd successfully auth them and try to redirect back to the original org page they tried to go to which would then take them back here.